translations/en-us.yml

```
@@ -1259,6 +1284,14 @@ registries:
         cancelButton: Cancel
         selectAll: 'Select All'
         clearAll: 'Clear All'
+    registrationList:
+        pending: 'No registrations pending approval have been found'
+        accepted: 'No accepted registrations have been found'
+        rejected: 'No rejected registrations have been found'
+        withdrawn: 'No withdrawn registrations have been found'
+        pending_withdraw: 'No registrations found pending withdrawal'
+        moderatorAction: '{action} {date} by {moderator}'
+        noActionsFound: 'Failed to load registration review history'
```

some consistency would be nice?
```suggestion
        moderator_action: '{action} {date} by {moderator}'
        no_actions_found: 'Failed to load registration review history'
```